servermaster/jobmanager.go

```diff
@@ -126,7 +129,11 @@ func (jm *JobManagerImplV2) InitImpl(ctx context.Context) error {
 
 // Tick implements lib.MasterImpl.Tick
 func (jm *JobManagerImplV2) Tick(ctx context.Context) error {
-	return nil
+	return jm.jobFsm.IterPendingJobs(
```

Comment:
tick will not be invoked,  tick is designed to be called by poll(in the  worker struct ), and the poll is called by the executor, not by server master

Reply:
`Poll` is called manually at https://github.com/hanfei1991/microcosm/blob/902e1bcd103f6b6af35c6a4905f0aa3d78e2fa55/servermaster/server.go#L508-L511